pages/common/echo.md

```diff
@@ -9,3 +9,11 @@
 - Print a message with environment variables:
 
 `echo {{"My path is $PATH"}}`
+
+- Print a message without the trailing newline:
+
+`echo -n {{"Hello World"}}`
+
+- Enable backslash escapes:
```

Comment:
How about "Enable enhanced mode to include special characters (escaped with backslash)"? That way the -e maps to "enhanced"